### PR TITLE
Feature/install auth

### PR DESCRIPTION
**Pull request by brad-Simeon**

### CODE REVIEW

**Review by rdorta-simeon — APPROVED** [review]

**Review by Lance-Simeon — DISMISSED** [review]

File: how-to.md [hunk 1]
```diff
@@ -27,6 +27,23 @@
 
 You can verify the licenses in your tenant [in the Azure Portal](https://portal.azure.com/#blade/Microsoft_AAD_IAM/ActiveDirectoryMenuBlade/Licenses) under **All products**
 
+## Tenant install options
+
+* **Delegated Authentication**: This option allows Simeon to read and write to the tenant with a user of your choosing (typically a pre-existing global administrator in the tenant or a user with the minimum [required permissions](https://simeoncloud.github.io/docs/#/permissions)). Simeon recommends using delegated authentication for all production tenants.
```
Comment (david-simeon) [inline how-to.md:32]: Global Administrator (capital G, capital A)

File: how-to.md [hunk 2]
```diff
@@ -27,6 +27,23 @@
...
+  * Simeon will create a refresh token for the user you authenticate with. This refresh token will cache the sign-in information for Simeon to use.
+  * You can authenticate with a user subject to MFA, Conditional Access, and other security policies.
+  * If you change the login credentials, MFA, or other sign-in policies for that user, you will need to log into the Simeon app and re-authenticate to create a new refresh token.
+  * Certain types of MFA enforcement can't be used with delegated authentication, such as location-based enforcement (unless using a self-hosted agent where you control the device location).
```
Comment (david-simeon) [inline how-to.md:36]: Let's try to avoid contractions, so let's say **cannot** instead of **can't**

File: how-to.md [hunk 3]
```diff
@@ -27,6 +27,23 @@
...
+
+* **Service Account**: This option creates a non-interactive Azure AD user with global administrator privileges to read and write in the tenant. Simeon recommends using the service account for non-production tenants where configurations frequently change, such as reference baseline tenants.
```
Comment (david-simeon) [inline how-to.md:38]: Global Administrator

Should also note that the user created is named simeon@tenantdomainname

File: how-to.md [hunk 4]
```diff
@@ -27,6 +27,23 @@
...
+  * You must exclude the service account from Conditional Access policies that restrict Simeon's access to the tenant.
```
Comment (david-simeon) [inline how-to.md:39]: You must exclude the service account from Conditional Access policies that restrict Simeon's access to the tenant, as this user must be excluded from MFA.

File: how-to.md [hunk 5]
```diff
@@ -27,6 +27,23 @@
...
+  * The service account does not require reauthentication after changing Conditional Access, MFA, or other policies.
```
Comment (david-simeon) [inline how-to.md:40]: Unless the policy change enforces MFA on the Simeon service account

File: how-to.md [hunk 6]
```diff
@@ -27,6 +27,23 @@
...
+
+* **Service Principal**: This option creates a service principal in the tenant to read and write configurations. You must select either delegated authentication or service account when using service principal.
+  * Microsoft does not support Syncing all configurations with a service principal. Wherever possible, Simeon will use the service principal to Sync configurations.
+  * When Syncing a configuration that is unsupported by the service principal, Simeon will use the user account for either delegated authentication or the service account, depending on what was selected when the tenant was installed.
```
Comment (david-simeon) [inline how-to.md:44]: Shorten second part to -- 

"Simeon will use the other selected option (service account or delegated authentication) to Sync the configuration..." 

or something like that

File: how-to.md [hunk 7]
```diff
@@ -27,6 +27,23 @@
...
+  * The service principal increases security for supported configurations as no user is involved.
```
Comment (david-simeon) [inline how-to.md:45]: Comma before as when as is functioning as "because"

File: how-to.md [hunk 8]
```diff
@@ -27,6 +27,23 @@
 
 You can verify the licenses in your tenant [in the Azure Portal](https://portal.azure.com/#blade/Microsoft_AAD_IAM/ActiveDirectoryMenuBlade/Licenses) under **All products**
 
+## Tenant install options
+
+* **Delegated Authentication**: This option allows Simeon to read and write to the tenant with a user of your choosing (typically a pre-existing Global Administrator in the tenant or a user with the minimum [required permissions](https://simeoncloud.github.io/docs/#/permissions)). Simeon recommends using delegated authentication for all production tenants.
+  * Simeon will create a refresh token for the user you authenticate with. This refresh token will cache the sign-in information for Simeon to use.
+  * You can authenticate with a user subject to MFA, Conditional Access, and other security policies.
+  * If you change the login credentials, MFA, or other sign-in policies for that user, you will need to log into the Simeon app and [reauthenticate](https://simeoncloud.github.io/docs/#/how-to?id=re-prompt-sync-to-complete-delegated-authentication) to create a new refresh token.
```
Comment (david-simeon) [inline how-to.md:35]: Change this to

If the refresh token is invalidated, then you will need to log into the Simeon app and [reauthenticate](https://simeoncloud.github.io/docs/#/how-to?id=re-prompt-sync-to-complete-delegated-authentication) to create a new refresh token. The token can be invalidated for reasons including changes to the login credentials, re-requiring MFA, and token expiration policies.

File: how-to.md [hunk 9]
```diff
@@ -27,6 +27,23 @@
...
+  * Certain types of MFA enforcement cannot be used with delegated authentication, such as location-based enforcement (unless using a self-hosted agent where you control the device location).
+
+* **Service Account**: This option creates a non-interactive Azure AD user named simeon@tenantdomainname with Global Administrator privileges to read and write in the tenant. Simeon recommends using the service account for non-production tenants where configurations frequently change, such as baseline tenants.
```
Comment (david-simeon) [inline how-to.md:38]: "with the Global Administrator role"

File: how-to.md [hunk 10]
```diff
@@ -27,6 +27,23 @@
...
+  * You must exclude the service account from Conditional Access policies that restrict Simeon's access to the tenant, as this user must be excluded from MFA.
+  * The service account does not require reauthentication after changing Conditional Access, MFA, or other policies (unless the policy change enforces MFA on the Simeon service account)
+
+* **Service Principal**: This option creates a service principal in the tenant to read and write configurations. You must select either delegated authentication or service account when using service principal.
+  * Microsoft does not support Syncing all configurations with a service principal. Wherever possible, Simeon will use the service principal to Sync configurations.
+  * When Syncing a configuration that is unsupported by the service principal, Simeon will use the user account for either delegated authentication or the service account.
```
Comment (david-simeon) [inline how-to.md:44]: When Syncing a configuration that is unsupported by the service principal, Simeon will use the user account for either delegated authentication or the service account.
-->
When Syncing a configuration that is unsupported by the service principal, Simeon will use the other option selected at install time - delegated authentication or the service account.

**Review by david-simeon — APPROVED** [review]

**Review by jeff-simeon — APPROVED** [review]

**Pull request merged**